server/sqldb/seed-dev.ts

```diff
@@ -45,17 +45,17 @@ export default (async function() {
 }());
```

Comment:
In this file, I just renamed `entityManager` to `manager` for consistency with the rest of the code